From c046a8bf6468fddbb8d320f27b1bd4dbd6d086b0 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Fri, 1 Oct 2010 13:57:53 +0400
Subject: [PATCH] No such option for which - '-s'

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>
---
 Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 2044bc9..2dfe104 100644
--- a/Makefile
+++ b/Makefile
@@ -23,13 +23,13 @@ INSTALLDIR = $(ERL_LIBS)/lfe
 all: compile docs
 
 compile:
-	if which -s rebar; \
+	if which rebar; \
 	then rebar compile; \
 	else $(ERLC) $(addprefix $(SRCDIR)/, $(SRCS)); \
 	fi
 
 install:
-	if which -s rebar; \
+	if which rebar; \
 	then rebar install; \
 	elif [ "$$ERL_LIBS" != "" ]; \
 	then mkdir -p $(INSTALLDIR)/$(EBINDIR) ; \
-- 
1.7.2.3

